extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -25,7 +25,7 @@ export const winPlatform: string = 'win32';
 export const macPlatform: string = 'darwin';
 export const linuxPlatform: string = 'linux';
 export const minSupportedNetCoreVersion: string = '3.1.0';
-export const maxSupportedNetCoreVersionCutoff: string = '6.0.0';	// un-set this to allow latest
+export const maxSupportedNetCoreVersionCutoff: string = '';	// un-set this to allow latest
```

Comment:
Why not just remove the code completely? You can add it back in later if/when you need to

Reply:
Is that better practice to remove it? It seems easier to just keep it so it's easy to add it back if/when build breaks again with a newer .net version.

Reply:
Given that we *just* released .NET6 I don't see .NET7 or whatever being anytime soon. And we have no idea about whether that'll even be a problem. So I'd definitely opt for "less unused code === better"

Reply:
true, the check for .Net 6 added a bunch of logic so it's better if we don't have to maintain it if it isn't being used.  I reverted the commit that added the .net 6 warning and fixed a couple of the merge conflicts.